server/src/main/java/org/elasticsearch/cluster/metadata/IndexReshardingMetadata.java

```diff
@@ -207,15 +208,22 @@ public static IndexReshardingMetadata newSplitByMultiple(int shardCount, int mul
         return new IndexReshardingMetadata(IndexReshardingState.Split.newSplitByMultiple(shardCount, multiple));
     }
 
-    public IndexReshardingMetadata transitionSplitTargetToHandoff(ShardId shardId) {
+    public static boolean isSplitTarget(ShardId shardId, @Nullable IndexReshardingMetadata reshardingMetadata) {
+        return reshardingMetadata != null && reshardingMetadata.isSplit() && reshardingMetadata.getSplit().isTargetShard(shardId.id());
```

Comment:
might be nice to get this covered in `IndexReshardingMetadataTests`. It could probably be three lines in the existing `testSplit`.